### PR TITLE
Accession date fields, refs #7764

### PR DESCRIPTION
**Pull request by mcantelon**
Related issues: #6526, #7290, #7764, #7905
- Renamed creation-related CSV import columns
- Access creation column support for accession CSV import
- Added related date support for accessions
- Added copying of accession's related dates when creation a description from an accession

### CODE REVIEW

File: lib/task/import/csvImportBaseTask.class.php [hunk 1]
```diff
@@ -94,4 +94,185 @@ protected function acceptsOption($name)
...
+        // Add creator history if specified
+        if(isset($import->rowStatusVars['creatorHistories'][$index]))
+        {
+          $eventData['actorHistory'] = $import->rowStatusVars['creatorHistories'][$index];
+        }
+
+        array_push($createEvents, $eventData);
+      }
+    }
+    else if(
```
Comment (MikeFE) [inline lib/task/import/csvImportBaseTask.class.php:225]: Minor style nitpick:
Please put spaces after your ifs and foreach loops. Also I'm not really sure why this if condition spans 4 lines, you could fit it in 2. There are several cases of this in the commit.

File: lib/task/import/csvImportTask.class.php [hunk 2]
```diff
@@ -1177,15 +1037,23 @@ function setupEventDateData(&$self, &$eventData, $index)
...
     {
       if (isset($self->rowStatusVars[$statusVar][$index]))
       {
-        $eventData[$eventProperty] = $self->rowStatusVars[$statusVar][$index];
+        if ($eventProperty == 'typeId')
+        {
+          $eventType = $self->rowStatusVars[$statusVar][$index];
+          $eventData[$eventProperty] = (strtolower($eventType) == 'accumulation') ? QubitTerm::ACCUMULATION_ID : QubitTerm::CREATION_ID;
+        }
+        else {
```
Comment (MikeFE) [inline lib/task/import/csvImportTask.class.php:1161]: Please put curly braces on a new line here like we do in the rest of AtoM

**Comment by sevein** [comment]
Is this ready? Let's do it!

**Comment by mcantelon** [comment]
Woot! I'll get the migration fixed for tomorrow (had to deal with some CVA
stuffs today).

Mike

On Thu, Mar 26, 2015 at 7:14 PM, Jesús García Crespo <
notifications@github.com> wrote:

> Is this ready? Let's do it!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/artefactual/atom/pull/106#issuecomment-86789364.

## 

Mike Cantelon ::  Open Source Development > LAMP / node.js / HTML5
http://mikecantelon.com

**Pull request closed without merging**